forge-gui/res/cardsfolder/upcoming/staff_of_titania.txt

```diff
@@ -0,0 +1,11 @@
+Name:Staff of Titania
+ManaCost:2
+Types:Artifact Equipment
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddPower$ X | AddToughness$ X | Description$ Equipped creature gets +X/+X, where X is the number of Forests you control. 
+T:Mode$ Attacks | ValidCard$ Creature.EquippedBy | Execute$ TrigToken | TriggerDescription$ Whenever equipped creature attacks, create a 1/1 green Forest Dryad land creature token. (It's affected by summoning sickness.)
+SVar:TrigToken:DB$ Token | TokenScript$ g_1_1_forest_dryad_land
```

Comment:
Looks like this token script isn't made yet

Reply:
https://github.com/Card-Forge/forge/pull/1784/files#diff-e3c2cd5d005981d2a00d57b8790d29a61baa99e7f0d50c28f93644bc2e447e4b